cmd/kata-manager/kata-manager.sh

```diff
@@ -0,0 +1,325 @@
+  disable-debug    : Turn off all debug options.
+  enable-debug     : Turn on all debug options for all system components.
+  install-packages : Install the packaged version of Kata Containers.
+  remove-packages  : Uninstall the packaged version of Kata Containers.
+  reset-config     : Undo changes to the runtime configuration [1].
+
+Notes:
+
+[1] - This is only possible if the user has not specified '-c'.
+
+
+EOT
+}
+
+die()
+{
+	local msg="$*"
+	echo "ERROR: $msg" >&2
+	exit 1
+}
+
+info()
+{
+	[ "$verbose" != "yes" ] && return
+
+	local msg="$*"
+	echo "INFO: $msg"
+}
+
+check_file()
+{
+	local file="$1"
+
+	[ -z "$file" ] && die "need file"
+
+	# Need to check as root to avoid EPERM which will cause the test to
+	# fail
+	sudo test ! -e "$file" && die "file '$file' does not exist" || true
+}
+
+config_checks()
+{
+	[ -e "$local_config_file" ] && return
+
+	# The user has specified a non-standard config file so the only way to
+	# handle this is to edit that file itself.
+	[ "$config_file" != "$local_config_file" ] && return
+
+	local dir=$(dirname "$local_config_file")
+
+	sudo mkdir -p "$dir"
+
+	# Create a copy of the default config file that this script can
+	# operate on, but only if that file doesn't already exist.
+	#
+	# Note: these permissions match the currently packaged permissions
+	if [ ! -e "$local_config_file" ]
+	then
+		sudo install -o root -g root -m 0644 \
+			"$default_config_file" \
+			"$local_config_file"
+
+		# use an image by default
+		cmd_configure_image "$default_image_file"
+	fi
+}
+
+disable_image()
+{
+	config_checks
+
+	sudo sed -i 's/^\(image *=.*\)/# \1/g' "$config_file"
+}
+
+disable_initrd()
+{
+	config_checks
+
+	sudo sed -i 's/^\(initrd *=.*\)/# \1/g' "$config_file"
+}
+
+enable_image()
+{
+	local file="$1"
+
+	config_checks
+
+	sudo sed -i "s!^#*.*image *=.*\$!image = \"$file\"!g" "$config_file"
+}
+
+enable_initrd()
+{
+	local file="$1"
+
+	config_checks
+
+	sudo sed -i "s!^#*.*initrd *=.*\$!initrd =\"$file\"!g" "$config_file"
+}
+
+cmd_enable_full_debug()
+{
+	info "enabling debug"
+
+	config_checks
+
+	sudo sed -i -e 's/^# *\(enable_debug\).*=.*$/\1 = true/g' "$config_file"
+	sudo sed -i -e "s/^kernel_params = \"\(.*\)\"/kernel_params = \"\1 ${agent_debug}\"/g" "$config_file"
+}
+
+cmd_disable_all_debug()
+{
+	info "disabling debug"
+
+	config_checks
+
+	sudo sed -i -e 's/^\(enable_debug.*=.*$\)/# \1/g' "$config_file"
+	sudo sed -i -e "s/^\(kernel_params = \".*\)${agent_debug}\(.*\"\)/\1 \2/g" "$config_file"
+}
+
+cmd_configure_image()
+{
+	local file="$1"
+
+	check_file "$file"
+
+	info "installing '$file' as image"
+
+	enable_image "$file"
+	disable_initrd
+}
+
+cmd_configure_initrd()
+{
+	local file="$1"
+
+	check_file "$file"
+
+	info "installing '$file' as initrd"
+
+	enable_initrd "$file"
+	disable_image
+}
+
+# Install the packaged version of Kata by executing the commands
+# specified in the installation guide document.
+cmd_install_packages()
+{
+	[ -z "${GOPATH}" ] && die "need GOPATH"
```

Comment:
`GOPATH` is now not a requirement to be set for `go` - maybe you can use a `go env GOPATH` here? - after maybe checking `go` is actually installed? ;-)

Reply:
Fixed.